samples/ar-view-node/src/main/java/io/github/sceneview/sample/arviewnode/nodes/TrackingNode.kt

```diff
@@ -0,0 +1,341 @@
+package io.github.sceneview.sample.arviewnode.nodes
+
+import android.content.Context
+import android.util.Log
+import android.view.View
+import android.widget.ImageView
+import androidx.core.content.ContextCompat
+import com.google.ar.core.DepthPoint
+import com.google.ar.core.Frame
+import com.google.ar.core.HitResult
+import com.google.ar.core.Plane
+import com.google.ar.core.Point
+import com.google.ar.core.Trackable
+import com.google.ar.core.TrackingState
+import com.google.ar.core.exceptions.DeadlineExceededException
+import com.google.ar.core.exceptions.FatalException
+import com.google.ar.core.exceptions.NotTrackingException
+import com.google.ar.sceneform.rendering.ViewAttachmentManager
+import com.google.ar.sceneform.rendering.ViewRenderable
+import dev.romainguy.kotlin.math.Float3
+import dev.romainguy.kotlin.math.Quaternion
+import io.github.sceneview.ar.ARSceneView
+import io.github.sceneview.ar.arcore.createAnchorOrNull
+import io.github.sceneview.ar.arcore.position
+import io.github.sceneview.ar.node.AnchorNode
+import io.github.sceneview.math.toRotation
+import io.github.sceneview.node.ViewNode
+import io.github.sceneview.sample.arviewnode.FrameTime
+import io.github.sceneview.sample.arviewnode.MainActivity
+import io.github.sceneview.sample.arviewnode.R
+
+
+class TrackingNode(
+    private val context: Context,
+    private val sceneView: ARSceneView
+) {
+
+    /**
+     * Indicates if the Camera is currently tracking.
+     */
+    private var isTracking = false
+
+    /**
+     * Indicates if the HitTest was successful.
+     */
+    private var isHitting = false
+
+    /**
+     * If we skip Frames this value is set to true, otherwise false.
+     */
+    private var isHittestPaused = true
+
+    private var createdCalled = false
```

Comment:
I'd rather call this flag `createCalled`.

Reply:
Check